triggers/tests/src/test/scala/com/digitalasset/daml/lf/engine/trigger/TriggerRuleSimulationLib.scala

```diff
@@ -777,8 +777,6 @@ final class TriggerRuleSimulationLib private (
             val msgIn = gb add TriggerContextualFlow[SValue].map(ctx =>
               ctx.copy(value = SList(FrontStack(ctx.value)))
             )
-            val encodeMsg =
-              gb add runner.encodeMsgs.map(ctx => ctx.copy(value = SList(FrontStack(ctx.value))))
```

Comment:
For this PR, this code is not needed - it will be added in on PR https://github.com/digital-asset/daml/pull/16507